Framework/Algorithms/src/ClearInstrumentParameters.cpp

```diff
@@ -35,55 +30,26 @@ const std::string ClearInstrumentParameters::category() const {
   return "DataHandling\\Instrument";
 }
 
-//----------------------------------------------------------------------------------------------
```

Comment:
We need to be careful here. My best guess is that this parameter was never explicitly set as part of a user script, unless @Anders-Markvardsen knows otherwise. Can you add something to the general release notes about this. I see you've already updated the algorithm documentation.